manim/mobject/text/text_mobject.py

```diff
@@ -730,7 +723,7 @@ def _get_settings_from_t2xs(
         return settings
 
     def _get_settings_from_gradient(
-        self, default_args: dict[str, Iterable[str]]
+        self, default_args: dict[str, str]
     ) -> Sequence[TextSetting]:
```

Comment:
In the same way you changed `Sequence` to `list` before, it should be possible to do the same here:
```suggestion
    ) -> list[TextSetting]:
```